src/v2/modules/contacts/controller.js

```diff
@@ -1,18 +1,17 @@
+'use strict';
+
 const Boom = require('@hapi/boom');
-const camelCaseKeys = require('../../../lib/camel-case-keys');
-const repositories = require('../../connectors/repository');
 
-const getContact = async request => {
-  const { contactId } = request.params;
-  const contact = await repositories.contacts.findOneById(request.params.contactId);
-  return contact ? camelCaseKeys(contact) : Boom.notFound(`No contact for ${contactId}`);
-};
+const { logger } = require('../../../logger');
+const contactService = require('../../services/contacts');
 
 const getContacts = async request => {
-  const { ids } = request.query;
-  const rows = await repositories.contacts.findManyById(ids.split(','));
-  return rows.map(camelCaseKeys);
+  try {
+    return contactService.getContactsByIds(request.query.ids.split(','));
```

Comment:
Do you know where we are using this endpoint in the water service? 
I think it would be much neater to have request.query.ids be an array which I think we did in another of the endpoints rather than a CSV.
I think then we might be able to use your new helper too.